extensions/wikia/WikiaApi/WikiaApiAjaxLogin.php

```diff
@@ -96,6 +96,10 @@ public function execute() {
 					$result['result'] = 'Throttled';
 					$result['text'] = wfMsg('login-throttled');
 					break;
+				case LoginForm :: ABORTED :
+					$result['result'] = 'Aborted';
+					$result['text'] = wfMsg($loginForm->mAbortLoginErrorMsg);
```

Comment:
True, makes sense. I must have seen wfMessage call in `UserLoginSpecialController` above.